lib/EditorJs.tsx

```diff
@@ -123,10 +124,18 @@ class EditorJsContainer extends React.PureComponent<Props> {
       })
   }
 
+  constructor(props: EditorJsProps) {
+    super(props)
+
+    this.holder = `editor-js-${(
```

Comment:
```suggestion
class EditorJsContainer extends React.PureComponent<Props> {
  holder = `editor-js-${...}`;
```

Why don't you try using class field? It's simple to initialize

Reply:
I've switched to that approach here: 71a71dc79a7cb59b882d75f7ffd10ba966c010b2